design/client.md

```diff
@@ -0,0 +1,12 @@
+# Client
+
+Since containerd uses gRPC, it should be easy to implement 3rd-party containerd client in almost any language.
+However, 3rd-party client would need to reimplement pull&push interactions with remote registries, as these are not implemented on the daemon side.
+
+## Label conventions
+
+Client implementations SHOULD follow this labeling convention so as to promote compatibility across different implementations.
+
+Label                              | Object                       | Description
+-----------------------------------|------------------------------|--------------------------------------------------
+`containerd.io/uncompressed`       | Blob (compressed layer)      | Digest of uncompressed layer blob.
```

Comment:
I think its weird to add documents like this for only one item.  Either do them all or none at all.  It just creates weird documents that are narrow and confuse the readers.

Reply:
Is there other item?

Using listing instead of table might be better?

Reply:
Is there other item?

Using listing instead of table might be better?

Reply:
I'll add GC labels after https://github.com/containerd/containerd/pull/1563